File: modul-2/PRAK104-2210817210012-MuhammadRakaAzwar.py
import math
a, b, c, d = 400000 , 350000, 0.13, 0.21
print("Harga sepatu A adalah", a)
print("Harga sepatu B adalah", b)
print("Sepatu A mendapat diskon 13% sehingga harganya menjadi",math.trunc(a-(a*c)));
print("Sepatu B mendapat diskon 21% sehingga harganya menjadi",math.trunc(b-(b*d)));

File: modul-2/PRAK104-2210817210012-MuhammadRakaAzwar.c
#include <stdio.h>
int main(){
float a = 400000 , b = 350000, c = 0.13, d = 0.21;
char diskon = '%';
printf("\Harga sepatu A adalah %.f\n", a);
printf("\Harga sepatu B adalah %.f\n", b);
printf("\Sepatu A mendapat diskon 13 %c sehingga harganya menjadi %.f\n", diskon, a-(a*c));
printf("\Sepatu B mendapat diskon 21 %c sehingga harganya menjadi %.f\n", diskon, b-(b*d));
}
